src/index.ts

```diff
@@ -23,11 +23,25 @@ type ServerlessResources = Service['resources'] & {
     Outputs?: Outputs;
 };
 
+interface ServerlessError extends Error {
+    code: string;
+}
+
+interface ServerlessErrorConstructor {
+    new (message?: string): ServerlessError;
+    (message?: string): ServerlessError;
+    readonly prototype: ServerlessError;
+}
+
+interface ServerlessClasses extends Serverless {
```

Comment:
Annoying you have to do all this.. maybe we should do a submission to the types repository.